community/hazelcast-jet-enterprise/values.yaml

```diff
@@ -0,0 +1,285 @@
+          </properties>
+          <edge-defaults>
+              <!-- capacity of the concurrent SPSC queue between each two processors -->
+            <queue-size>1024</queue-size>
+              <!-- network packet size limit in bytes, only applies to distributed edges -->
+            <packet-size-limit>16384</packet-size-limit>
+              <!-- receive window size multiplier, only applies to distributed edges -->
+            <receive-window-multiplier>3</receive-window-multiplier>
+          </edge-defaults>
+          <!-- whether metrics collection is enabled -->
+          <metrics enabled="true">
+              <!-- the number of seconds the metrics will be retained on the instance -->
+              <retention-seconds>120</retention-seconds>
+              <!-- the metrics collection interval in seconds -->
+              <collection-interval-seconds>5</collection-interval-seconds>
+              <!-- whether metrics should be collected for data structures. Metrics
+                  collection can have some overhead if there is a large number of data
+                  structures -->
+              <metrics-for-data-structures>false</metrics-for-data-structures>
+          </metrics>
+      </hazelcast-jet>
+      
+# nodeSelector is an array of Hazelcast Jet Node labels for POD assignments
+# ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+nodeSelector: {}
+
+gracefulShutdown:
+  enabled: true
+  maxWaitSeconds: 600
+
+# Hazelcast Liveness probe
+livenessProbe:
+  # enabled is a flag to used to enable liveness probe
+  enabled: true
+  # initialDelaySeconds is a delay before liveness probe is initiated
+  initialDelaySeconds: 30
+  # periodSeconds decides how often to perform the probe
+  periodSeconds: 10
+  # timeoutSeconds decides when the probe times out
+  timeoutSeconds: 5
+  # successThreshold is the minimum consecutive successes for the probe to be considered successful after having failed
+  successThreshold: 1
+  # failureThreshold is the minimum consecutive failures for the probe to be considered failed after having succeeded
+  failureThreshold: 3
+
+# Hazelcast Readiness probe
+readinessProbe:
+  # enabled is a flag to used to enable readiness probe
+  enabled: true
+  # initialDelaySeconds is a delay before readiness probe is initiated
+  initialDelaySeconds: 30
+  # periodSeconds decides how often to perform the probe
+  periodSeconds: 10
+  # timeoutSeconds decides when the probe times out
+  timeoutSeconds: 1
+  # successThreshold is the minimum consecutive successes for the probe to be considered successful after having failed
+  successThreshold: 1
+  # failureThreshold is the minimum consecutive failures for the probe to be considered failed after having succeeded
+  failureThreshold: 3
+
+# Configure resource requests and limits
+# ref: http://kubernetes.io/docs/user-guide/compute-resources/
+#
+resources:
```

Comment:
Exposing resources in this way forces a user to provide values for all 4 of these settings, which may not be what they want to do.  A better way is probably to just expose it as `resources: {}` and let the user specify which settings and values they want to use.  See these links for examples.
https://github.com/IBM/charts/blob/master/stable/ibm-istio-remote/values.yaml#L161
https://github.com/IBM/charts/blob/master/stable/ibm-istio-remote/values-metadata.yaml#L350